frontend/src/communications/userApi.js

```diff
@@ -1,5 +1,9 @@
 import axios from "axios";
 
 export const login = (value) => {
-   return axios.post("http://localhost:3000/api", value)
-}
+  return axios.post("http://localhost:3000/api", value);
```

Comment:
Lehetne axios instance-t csinálni és akkor nem kellene mindig beírni a hívásokba a domain-t meg a portot. Lásd: https://github.com/axios/axios#creating-an-instance